pkgs/applications/office/docear/default.nix

```diff
@@ -25,7 +25,7 @@ stdenv.mkDerivation {
     # The wrapper ensures oraclejre is used
```

Comment:
This comment is obsolete.

Reply:
Removed in latest push; it looks self-explanatory.